src/main/java/testUI/Utils/AppiumHelps.java

```diff
@@ -301,23 +301,23 @@ public static void sleep(long milliseconds) {
     private static List getMobileElementList(String locator) {
```

Comment:
By adding specific `List` type, you don't have to provide specific type cast, update all this function usages accordingly
```suggestion
    private static List<WebElement> getMobileElementList(String locator) {
```